src/packet/sym_encrypted_session_key.js

```diff
@@ -105,6 +97,11 @@ class SymEncryptedSessionKeyPacket {
     }
   }
 
+  /**
+   * Create a string representation of a tag 3 packet
```

Comment:
```suggestion
   * Create a binary representation of a tag 3 packet
```
I know you copied this from somewhere that has it wrong too, feel free to fix it there too, if you feel like

Reply:
Yeah, I copied it without really thinking if it made sense in the original use. Fixed in both places.